fares-data-build-tool/fdbt-site
# Redirect to correct page if one product

# Description

- Redirect to correct page for one product
- some cleaning up of db types
- throw error if no stops in matching json

# Testing instructions

- Navigate to the howManyProducts page
- Enter 1
- You should be redirected to the periodProduct page
- Ensure all journeys are still working

# Type of change


# Checklist:

-   [ ] Lighthouse performed - N/A
